src/main/java/io/vertx/core/http/impl/WebSocketImplBase.java

```diff
@@ -349,6 +371,18 @@ public S binaryMessageHandler(Handler<Buffer> handler) {
     }
   }
 
+  @Override
+  public WebSocketBase pongHandler(Handler<Buffer> handler) {
+    synchronized (conn) {
+      checkClosed();
+      if (frameHandler == null || frameHandler.getClass() != FrameAggregator.class) {
```

Comment:
since pong frames are always final and not aggregated, I believe we don't need a frame aggregator

Reply:
doh.  I should have moved the variable to the class level.